src/app/qgisapp.cpp

```diff
@@ -4490,6 +4490,15 @@ void QgisApp::about()
     versionString += "</tr><tr><td colspan=4>" + tr( "This copy of QGIS writes debugging output." ) + "</td>";
 #endif
 
+#ifdef WITH_BINDINGS
+    if ( mPythonUtils && mPythonUtils->isEnabled() )
+    {
+      versionString += "</tr><tr><td>" + tr( "Active python plugins" ) + "</td><td colspan=3>";
+      const QStringList activePlugins = mPythonUtils->listActivePlugins();
+      versionString += activePlugins.join( "; <br>" ) + "</td>";
```

Comment:
Actually, what's the `;` for? I think they should either be `;` separated or new line separated

Reply:
The ;s are there to separate plugins when pasted in github (the html to markdown parsing kills new lines). 

New lines are because it makes for an easier list to visually glance over in the about dialog.